charts/posthog/values.yaml

```diff
@@ -674,6 +674,56 @@ clickhouse:
     # prometheus.io/scrape: "true"
     # prometheus.io/path: /metrics
     # prometheus.io/port: "9363"
+  
+  client:
+    image:
+    # -- Clickhouse client image repository.
+      repository: yandex/clickhouse-client
+      # -- ClickHouse backup image tag.
+      tag: "21.3.20.1"
+
+  backup:
+    # https://posthog.com/docs/self-host/runbook/clickhouse/backup
+    # https://github.com/AlexAkulov/clickhouse-backup
+    enabled: true
+    image:
+      # -- Clickhouse backup image repository.
+      repository: altinity/clickhouse-backup
+      # -- ClickHouse backup image tag.
+      tag: "1.4.0"
+    backup_user: backup
+    # PASSWORD=backup_password; echo "$PASSWORD"; echo -n "$PASSWORD" | sha256sum | tr -d '-'
+    backup_password: eb94c11d77f46a0290ba8c4fca1a7fd315b72e1e6c83146e42117c568cc3ea4d
```

Comment:
It looks like this is expected to be sha256, used for `password_sha256_hex` in the config, but also for BACKUP_PASSWORD env. Does this work? I would have thought we'd need to use the plain text password for BACKUP_PASSWORD? I haven't tested this so maybe it works, have you got it to work @navi86 ?

Reply:
@hazzadous 
yes, it works, I have a backup that allows me to restore from it.

Reply:
Note that the admin password isn't sha256 encoded there though, and we use password, not password_sha265_hex when configuring ClickHouse:

https://github.com/PostHog/charts-clickhouse/pull/383/files#diff-810e35d3f26851e3c05e51f65600b5d393ff06eb53b85d9e0c22a5918ae0d4deR9

Anyway, working trumps conjecture

Reply:
@hazzadous 
I took manifest from 
https://github.com/AlexAkulov/clickhouse-backup/blob/master/Examples.md#how-to-use-clickhouse-backup-in-kubernetes
and it worked for me